packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -13,15 +13,19 @@
                 :src="card.image"
                 :alt="card.title">
             <h3
+                v-make-text-accessible
```

Comment:
I haven't used directives at all before – this is v cool  🚀

Reply:
It is cool, but I only use them when we have ruled out other options generally.